demos/guide/build.info

```diff
@@ -29,3 +30,7 @@ DEPEND[tls-client-non-block]=../../libcrypto ../../libssl
 INCLUDE[quic-client-non-block]=../../include
 SOURCE[quic-client-non-block]=quic-client-non-block.c
 DEPEND[quic-client-non-block]=../../libcrypto ../../libssl
+
+INCLUDE[quic-client-non-block]=../../include
+SOURCE[quic-client-non-block]=quic-server-block.c
+DEPEND[quic-client-non-block]=../../libcrypto ../../libssl
```

Comment:
looks like copy'n'paste typo which might be relevant to failures I see.
s/quic-client/quic-server